packages/dds/merge-tree/src/test/client.applyMsg.spec.ts

```diff
@@ -1015,6 +1015,48 @@ describe("client.applyMsg", () => {
 
 			logger.validate({ baseText: "BBBBBB B" });
 		});
+		// TODO: see AB#29398
+		it.skip("obliterate with mismatched final states", () => {
+			const clients = createClientsAtInitialState(
+				{
+					initialState: "B{6666666}BBB{666666}BB",
+					options: {
+						mergeTreeEnableObliterate: true,
+						mergeTreeEnableSidedObliterate: true,
+						mergeTreeEnableAnnotateAdjust: true,
+					},
+				},
+				"A",
+				"B",
+			);
+			let seq = 0;
+			const logger = new TestClientLogger(clients.all);
+			const ops: ISequencedDocumentMessage[] = [];
+			const b = clients.B;
+
+			ops.push(
+				// no way to know if annotates are regular or adjust
```

Comment:
I don't understand this comment. Looks like it's regular, not adjusted. Seems like adjust option is not needed. And it wouldn't be used on obliterated ranges [for now].

Reply:
the type of annotate really shouldn't matter. i'm surprised the repo needs an annotate at all, but it could make sense if the case needs some kind of segment boundary, as annotate does split segments., but beyond that really have no other interactions with the tree structure

Reply:
This was left in by mistake, but what I meant was that I was unsure if the farm test applied a regular or adjusted annotate (since I don't think they're treated differently in the test logger). Not sure if it matters though, so I went ahead and made the op for this test a regular annotate.

Reply:
Turns out that the annotate isn't required for the test to fail, so I removed it to have one less op to keep track of.